pkg/cmd/util/pluginconfig/config.go

```diff
@@ -2,11 +2,30 @@ package pluginconfig
 
 import (
 	"io/ioutil"
+	"os"
+
+	"k8s.io/apimachinery/pkg/apimachinery/announced"
+	"k8s.io/apimachinery/pkg/apimachinery/registered"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	kapiserverinstall "k8s.io/apiserver/pkg/apis/apiserver/install"
+	kapiserverv1alpha1 "k8s.io/apiserver/pkg/apis/apiserver/v1alpha1"
 
 	configapi "github.com/openshift/origin/pkg/cmd/server/api"
 	"github.com/openshift/origin/pkg/cmd/server/api/latest"
 )
 
+var (
+	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
+	registry             = registered.NewOrDie(os.Getenv("KUBE_API_VERSIONS"))
```

Comment:
I don't want to respect this env var.  I don't think it was a good idea and realistically people can't safely set it.